src/foam/u2/ExportModal.js

```diff
@@ -247,15 +247,10 @@ foam.CLASS({
           var link = document.createElement('a');
           var href = '';
           if ( self.exportDriverReg.mimeType && self.exportDriverReg.mimeType.length != 0 ) {
-            var prefix = 'data:' + self.exportDriverReg.mimeType + ',';
-            href = encodeURI(prefix + result);
-          } else {
-            href = result;
-          }
-
-          if ( href.length > 100000 ) {
             var blob = new Blob([result], { type: self.exportDriverReg.mimeType });
             href = URL.createObjectURL(blob);
+          } else {
+            console.error("Data type for export not specified");
```

Comment:
This is misleading if the length is zero.

Reply:
Empty string == not specified, isn't it? What would be better message?

Also, changed console.error to throw new Error, so user will not start downloading of broken file

Reply:
Ah, sorry, I was reading the length != 0 as a test of the data set, not the mime type name.